### ISSUE TITLE
Seeing CORS issues on Chrome Browser

### ISSUE THREAD

**Issue by mram0509** [kevinselwyn/fast#2]
Opening another issue as the older CORS related issue is closed.

I am viewing index.html in a server environment and still getting the issue. What's the fix for this? I am using Chrome 72.0.3626
Running the code as is I get the following error:
Access to XMLHttpRequest at 'http://api.fast.com/netflix/speedtest?https=true&token=YXNkZmFzZGxmbnNkYWZoYXNkZmhrYWxm&urlCount=3' from origin 'http://www.sample.com' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'null' that is not equal to the supplied origin.

I tried adding this to the xhr object:
xhr.setRequestHeader("Access-Control-Allow-Origin", "*");

Now the error has to do with pre-flight issues:
index.html:1 Access to XMLHttpRequest at 'http://api.fast.com/netflix/speedtest?https=true&token=YXNkZmFzZGxmbnNkYWZoYXNkZmhrYWxm&urlCount=3' from origin 'http://www.sample.com' has been blocked by CORS policy: Request header field access-control-allow-origin is not allowed by Access-Control-Allow-Headers in preflight response.

**Comment by kevinselwyn** [comment]
Good catch! After some digging it looks like the `http://api.fast.com` domain is whitelisted for CORS when the server is running on port 8000 aka `http://localhost:8000`. I've updated the README for this repo. Try that and let me know if it also works for you.

**Comment by mram0509** [comment]
running on port 8000 works!
